package.json

```diff
@@ -1599,6 +1600,7 @@
                     "items": {
                         "type": "string",
                         "enum": [
+                            "Compose Project Name",
```

Comment:
Many of the other settings that are multiple words don't have spaces, do we know why that is? e.g. `RepositoryNameAndTag`

Reply:
We had a chat about this. Most of the property names are coming from docker and hence with no space. "Compose Project Name" is not exactly a docker property, we derive this value from a label.

Reply:
Gotcha, makes sense.